crates/proof-of-sql/src/sql/transform/owned_table_postprocessing.rs

```diff
@@ -0,0 +1,9 @@
+use crate::base::{database::OwnedTable, scalar::Scalar};
+use std::fmt::Debug;
+
+/// A trait for nodes that can apply transformations to a `OwnedTable`.
+pub trait OwnedTablePostprocessing<S: Scalar>: Debug + Send + Sync {
```

Comment:
I wonder if this would be better as an enum.

Reply:
I was wondering as well haha.